### PR TITLE
Apply Modal Component and Dynamically Generate Previous Works Contents

### PR DESCRIPTION
**Pull request by adamilare**
In this excersise, following features were implemented into the existing Portfolio Project;
- Dynamically generate previous works card components using Javascript
   - Use array of objects that hold details of each project.
   - Iterate through the array to create the work cards
- Create modal component that opens when a previous work is clicked for detail.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -94,174 +94,7 @@ <h1>
...
-                A daily selection of privately personalized reads; no accounts
-                or sign-ups required. has been the industry's standard
-              </p>
-              <ul class="work-stack">
-                <li class="stack-item">html</li>
-                <li class="stack-item">bootstrap</li>
-                <li class="stack-item">Ruby</li>
-              </ul>
-            </article>
-          </div>
```
Comment (emmanuellekamwa) [inline index.html:264]: - [ ] When I click on a project I don't see any pop-up. Please try and fix that.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ❌⛔**
Hello @adamilare 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- You documented your work professionally.✔
- Your linters are passing all the checks.

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (adamilare) [reply index.html:264]: @emmanuellekamwa I just re-cloned now to test out the issue pointed out and the pop-up showed. 
Could please provide more details on what could be wrong?

Did you preview from the modal branch?

Please I need more insight into what the issue is.

Thank you for your review

Comment (emmanuellekamwa) [reply index.html:264]: I have tested your project on the live demo link that is on your readme and I cloned your project too but the popup is not still showing.

Comment (adamilare) [reply index.html:264]: Live demo is served from the main branch, not this feature branch. And to really test out this pull request, it has to be tested based on the content of the "modal" branch. 

From your clone copy please checkout to modal branch and preview from their!

Comment (adamilare) [reply index.html:264]: Besides, it is the "See Project" that triggers the pop-up, no the whole card

Comment (emmanuellekamwa) [reply index.html:264]: Just request for another review.

Comment (adamilare) [reply index.html:264]: I need to know what is wrong with this first, I cannot just waste a review.
Hope you understand my concern.

Comment (emmanuellekamwa) [reply index.html:264]: Everything is working now.Could you just request another review?

**Review by Deepakdanger — APPROVED** [review]
Hi @adamilare , 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**